create.go

```diff
@@ -72,6 +72,10 @@ func create(containerID, bundlePath, console, pidFilePath string) error {
 		return err
 	}
 
+	if pidFilePath == "" {
```

Comment:
This one isn't necessary. We check it already in 

```
func createPIDFile(pidFilePath string, pid int) error {
	if pidFilePath == "" {
		return errNeedPidFilePath
	}
```

I would also argue that second one totally isn't necessary either. One of the functions using pidFilePAth will return an error.